services/audit/README.md

```diff
@@ -0,0 +1,13 @@
+# Audit service
+
+The audit service logs all events of the system as an audit log. Per default, it will be logged to standard out, but can also be configured to a file output. Supported log formats are json or a simple key-value pair ("key1=value1 key2=value2").
```

Comment:
```suggestion
The audit service logs all events of the system as an audit log. Per default, it will be logged to standard out, but can also be configured to a file output. Supported log formats are json or a simple key-value pair like:

file_delete)
   user 'user_id' trashed file 'item_id'
file_trash_delete)
   user 'user_id' removed file 'item_id' from trashbin
```
damn, I cant add triple quotes into the suggestion example 😠 

do you have a json example too?

Reply:
I will continue locally and push up again with changes